### DISCUSSION TITLE
about getting gladcam data

### DISCUSSION THREAD

**Discussion by zeynepozdemir** [open-mmlab/mmclassification#1364] [Q&A]
I got an error like this when trying to run the gladcam module in the mmclassification library. Can you help me?

![image](https://user-images.githubusercontent.com/6259137/218062390-e4101a71-a39c-471e-8add-e16917a73124.png)

**Comment by Ezra-Yu** [comment]
I am sorry I can not reproduce your error as I run in the latest `dev-1.x` branch
```
python tools/visualizations/vis_cam.py \
    demo/bird.JPEG \
    configs/resnet/resnet50_8xb32_in1k.py \
    https://download.openmmlab.com/mmclassification/v0/resnet/resnet50_batch256_imagenet_20200708-cfb998bf.pth \
    --method GradCAM
```

![image](https://user-images.githubusercontent.com/18586273/218071284-a8aacf07-e409-4d1a-93f8-88d4b9ff4311.png)
and get 
![image](https://user-images.githubusercontent.com/18586273/218071758-0cdcfb4a-7c0e-4805-b0e5-d52c129a4e4f.png)




What platform and what version did you have the problem on?
